### ISSUE TITLE
My Tesla app was working fine. Now, there is no vehicle data, nor can I choose a vehicle. I have deleted the app, reinstalled all devices, restarted my phone and watch.  Still nothing. Please help. 

### ISSUE THREAD

**Issue by marclebo1** [Kemmey/Kemmey-TeslaWatch-Public#325]

**Comment by marclebo1** [comment]
Yes, the vehicle shows in the official app.  I bought the car one month ago.  For about 10 days it worked fine.  Then it stopped working.  Here is the screen.  When you hit choose vehicle, the next screen is totally blank. What now.  Thanks

**Comment by Kemmey** [comment]
@marclebo1 Ok, if the vehicle chooser screen is just completely blank, it sounds like the apps not getting updated data from Tesla - which really should log you out or do something. Cause if there's no vehicles on your account from an api perspective, the app should show "no vehicles on account" on the front screen. 

Two things

1. Please go to settings in the app on your phone and log out, then log back in - see if that makes any difference. It should clear all stored data, which doesn't necessarily happen when just removing the app. 
2. Go to settings, scroll to bottom, tap show debug, scroll to bottom and then tap the Email logs button, and send me the logs - I'll see if I can spot anything weird.

**Comment by marclebo1** [comment]
Hi, just sent to you...Marc

**Comment by Kemmey** [comment]
Thanks - got the logs!

Definitely something super weird going on - the login is working just fine, as your token is properly being generated and used - but the requests to the Tesla api are failing with a currently unknown error. 

Are you by any chance connected through any sort of vpn service?

I'd like you to join the public beta so I can add some more logging around this unknown error and get some insight into what shenanigans Tesla are pulling - would you be up for that?

If so, please follow the instructions at this address - from your phone: 

https://testflight.apple.com/join/FdzFOLZv

**Comment by marclebo1** [comment]
No vpn and yes to the public beta...

**Comment by marclebo1** [comment]
Just joined the beta...

**Comment by Kemmey** [comment]
If you open the TestFlight app you should see an update. Please update, then try logging out and back in, and send me the logs again.

**Comment by marclebo1** [comment]
Just sent...

**Comment by Kemmey** [comment]
@marclebo1 

This is the result I get from the api from the logs:

8:09 AM: Got vehicles response: {"response":[],"count":0}￼

So Tesla reports no vehicles on account. Nothing I can do about that. 

Try logging into your account on Tesla.com and check if your vehicle is attached - if not, contact Tesla support. 

I would be wary of logging out of the Tesla app, as it might not work after logging back in.

**Comment by marclebo1** [comment]
Ok I’ll try

**Comment by marclebo1** [comment]
Ok, so I’m working with Tesla support to merge my account with my wife’s, who is apparently the main.  They said it would take a day or so.  I’m hoping you cracked the case.  I’m not sure how the app initially worked, but we’ll wait and see.  Thanks for your continued efforts!!!

**Comment by Kemmey** [comment]
In the meantime, try logging in with your wife's account? Which account is logged in really doesn't affect use - automatic profile switching in car is entirely based on phone hardware, not logged in account :-D 

We just share mine!

**Comment by marclebo1** [comment]
I was just able to log in on my wife’s account and on my iPhone all is good!!!

However, on my watch it’s trying to load, but will not.  Any ideas?  We’re so close...

**Comment by marclebo1** [comment]
Kim - it just loaded on watch!  Thank you, thank you thank you...
